android/app/src/main/res/layout/onboarding_slide.xml

```diff
@@ -24,7 +24,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:textColor="@color/colorPrimaryDark"
-            android:fontFamily="@font/jost_medium"
+            android:fontFamily="@font/jost_light"
```

Comment:
Ideally, the changes to this file would be in a different PR - but since they already appear to be in a dedicated commit, as long as we don't squash merge this, it's fine by me.